scene/gui/control.cpp

```diff
@@ -49,6 +49,7 @@
 #include "servers/text_server.h"
 
 #ifdef TOOLS_ENABLED
+#include "editor/editor_settings.h"
```

Comment:
I see it's a pre-existing issue here, but `scene` code shouldn't depend on `editor` code.

IMO the `root_dir` stuff should be made a public (static?) property and set from outside `control.cpp` by the editor. Similar to how we handle swapping Ok/Cancel on AcceptDialog.

Reply:
Changed to use static property.